tests/test_vrs_normalize.py

```diff
@@ -35,27 +35,65 @@
     }
 }
 
-seq_loc = {
+
+allele_dict2_normalized = {
+    "type": "Allele",
+    "location": {
         "type": "SequenceLocation",
-        "sequence": "refseq:NC_000001.11",
-        "start": [None, 244988599],
-        "end": [244988601, None]
+        "sequence": "refseq:NC_000023.11",
+        "start": [None, 155980375],
+        "end": [155980377, None]
+    },
+    "state": {
+        "length": 0,
+        "repeatSubunitLength": 2,
+        "type": "ReferenceLengthExpression"
     }
+}
+
 
 allele_dict3 = {
     "type": "Allele",
-    "location": seq_loc,
+    "location": {
+        "type": "SequenceLocation",
+        "sequence": "refseq:NC_000023.11",
+        "start": [155980374, 155980375],
+        "end": [155980377, 155980378]
+    },
     "state": {
-        "type": "RepeatedSequenceExpression",
-        "seq_expr": {
-            "location": seq_loc,
-            "type": "DerivedSequenceExpression",
-            "reverse_complement": False
-        },
-        "count": {
-            "type": "Number",
-            "value": 2
-        }
+        "sequence": "",
```

Comment:
This is acknowledging that the entire concept of an Allele (and not a CopyNumber) with ambiguous endpoints is a little absurd. I wonder if we should even be supporting that now that we do CNC / CNX.

Reply:
It's absurd because I added these tests over 2 years ago (when I knew almost nothing). If @ahwagner and @larrybabb  could provide some good test examples, that would be great 😄

Reply:
Sorry @korikuzma; I didn't mean to indicate that the test was absurd, the notion that @larrybabb and I have not disallowed this use case is absurd. The tests are a good reflection of what we have asked for.